app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class UsersController < ApplicationController
   def index
-    @users = User.order(:id).page(params[:page])
+    @users = User.order(:id).page(params[:page]).with_attached_icon
```

Comment:
```suggestion
    @users = User.with_attached_icon.order(:id).page(params[:page])
```

orderやページネーションのメソッドが最後の方に来ることが慣習的に多いかなと思います。